distrobox-init

```diff
@@ -947,6 +947,8 @@ if [ "${nvidia}" -eq 1 ]; then
 	# because we need to extract the relative path to mount them in the
 	# correct path based on the guest's setup
 	NVIDIA_LIBS="$(find /run/host/usr/lib* -iname "*nvidia*.so*" || :)"
```

Comment:
We can just add `-o -iname "*cuda*.so*"` to this command and all will be ok :+1:

Reply:
Feel free to go ahead and make the fix, just putting this out there as an example :) Not sure I would link all host libraries into the container though. There might be distro specific differences